backend/src/main/java/com/fortunator/api/controller/FinancialMovementsController.java

```diff
@@ -0,0 +1,34 @@
+package com.fortunator.api.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fortunator.api.controller.entity.FinancialMovement;
+import com.fortunator.api.controller.entity.MovementByCategory;
+import com.fortunator.api.service.FinancialMovementsService;
+
+@RestController
+@RequestMapping(value = "/financial-movements")
```

Comment:
Geralmente em APIs rest a galera usa underscore ao invés de traço. O que acha ? 

```suggestion
@RequestMapping(value = "/financial_movements")
```